src/web/releases.rs

```diff
@@ -244,57 +244,94 @@ fn get_releases_by_owner(
     (author_name, packages)
 }
 
+/// Get the search results for a search query
+///
+/// Retrieves crates which names have a levenshtein distance of less than or equal to 3,
+/// crates who fit into or otherwise are made up of the query or crates who's descriptions
+/// match the search query.
+///
+/// * `query`: The query string, unfiltered
+/// * `page`: The page of results to show (1-indexed)
+/// * `limit`: The number of results to return
+///
+/// Returns `None` if no results are found and `Some` with the total number of results and the
+/// currently requested results
+///
 fn get_search_results(
     conn: &Connection,
-    query: &str,
+    mut query: &str,
     page: i64,
     limit: i64,
 ) -> Option<(i64, Vec<Release>)> {
+    query = query.trim();
+    let split_query = query.replace(' ', " & ");
```

Comment:
What does this do?

Reply:
Ah, the first one trims leading/trailing whitespace and the second makes a query that's ready to be made into a `ts_query`

Reply:
Let's use `plainto_tsquery` instead which removes the need to normalize spaces and also gets rid of words like 'the'.

https://www.postgresql.org/docs/9.5/textsearch-controls.html